theme.css

```diff
@@ -1814,7 +1825,7 @@ ul.comparison-list > li em {
 
 /** Insights graphs */
 .graph-canvas .axis .tick text {
-  fill: var(--black-1)
+  fill: var(--black-1)  
 }
 
 .graph-canvas .axis line {
```

Comment:
Could you update this style to this?  The change of the card content to white-2 makes the lines a little hard to read so this will improve that.

```css
.graph-canvas .axis line {
  stroke: var(--white-3);
}
```